react/AddressSummary.tsx

```diff
@@ -44,11 +45,15 @@ const AddressSummaryWithAddress: React.VFC = () => {
 
   const addressRules = useAddressRules()
 
+  if (selectedAddress != null && addressRules == null) {
+    return <Loading />
+  }
```

Comment:
the reason I check for the selected address here as well is so I don't show a loading state if there isn't any address to "format" (using the address rules), which would result in a awkward experience